avaframe/runScripts/runD2Th.py

```diff
@@ -33,7 +33,7 @@ def runD2Th(avaDir, comMod, resType, profileAxis, profileIndex):
         # convert depth to thickness using dem
         thicknessDict, depthRasterResized, slopeAngleField = tF.convertDepthToThickness(depthField, dem)
 
-        pName = rF.stem.split("_%s" % resType)[0] + "transformed" + "_%s.asc" % resType
+        pName = rF.stem.split("_%s" % resType)[0] + "transformed" + "_%s" % resType
```

Comment:
shall we add ... + "_%s." % resType - so add the point?

Reply:
misunderstanding